src/components/pages/career/application-process.tsx

```diff
@@ -24,13 +24,17 @@ const AccordionText = styled.p`
   margin-bottom: -3px;
 `;
 
+const ApplicationWrapper = styled.div`
```

Comment:
I would name it `ApplicationProcessWrapper` so it doesn't sound ambiguous

Reply:
great suggestion! Thanks, I have changed it too.